Docs: add server view folder (#30849)
grafana/grafana: docs/sources/administration/view-server/view-server-settings.md

title = "View server settings"
description = "How to view server settings in the Grafana UI"
keywords = ["grafana", "configuration", "server", "settings"]
aliases = ["/docs/grafana/latest/admin/view-server-settings/"]
weight = 300
+++

# View Grafana server settings

> If you are a Grafana server admin, then you can use the Settings tab to view the settings applied to your Grafana server by the [Configuration]({{< relref "../configuration.md#config-file-locations" >}}) file and any environmental variables.

Reviewer: ```suggestion
If you are a Grafana server administrator, use the Settings tab to view the settings that are applied to your Grafana server via the [Configuration]({{< relref "../configuration.md#config-file-locations" >}}) file and any environmental variables.
```

> > **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).

Reviewer: ```suggestion
> **Note:** Only Grafana server administrators can access the **Server Admin** menu. For more information about about administrative permissions, refer to [Grafana server admin]({{< relref "../../permissions/_index.md" >}}).
```
